mne/cov.py

```diff
@@ -1431,11 +1436,12 @@ def _get_whitener_data(info, noise_cov, picks, diag=False, rank=None,
         noise_cov = cp.deepcopy(noise_cov)
         noise_cov['data'] = np.diag(np.diag(noise_cov['data']))
 
-    scalings_ = dict(mag=1e12, grad=1e11, eeg=1e5)
+    _scalings = DEFAULTS['scalings_cov_rank']
     if isinstance(scalings, dict):
-        scalings_.update(scalings)
+        _scalings.update(scalings)
+    scalings = _scalings
 
-    W = compute_whitener(noise_cov, info, rank=rank, scalings=scalings_)[0]
+    W = compute_whitener(noise_cov, info, rank=rank, scalings=scalings)[0]
```

Comment:
For <code>_estimate_rank_meeg_signals</code> (line 1722) and <code>_estimate_rank_meeg_cov</code> (line 1771).